pkg/volume/awsebs/aws_ebs_block.go

```diff
@@ -66,6 +66,8 @@ func (plugin *awsElasticBlockStorePlugin) getVolumeSpecFromGlobalMapPath(volumeN
 	}
 	fullVolumeID := strings.TrimPrefix(globalMapPath, pluginDir) // /vol-XXXXXX
 	fullVolumeID = strings.TrimLeft(fullVolumeID, "/")           // vol-XXXXXX
+	// Windows paths have \\ instead.
+	fullVolumeID = strings.TrimLeft(fullVolumeID, "\\") // vol-XXXXXX
```

Comment:
do we need to check whether it is windows os?

Reply:
Hm, we could check... although it shouldn't be necessary, unless the ``fullVolumeId`` would have some leading backslashes for some reason. But better safe than sorry.